com.arc.embeddedcdt/src/com/arc/embeddedcdt/launch/Launch.java

```diff
@@ -533,6 +536,11 @@ private void start_ashling(final ILaunchConfiguration configuration, final ILaun
                 DebugPlugin.exec(DebugPlugin.parseArguments(ash_cmd), ash_dir),
                 ASHLING_PROCESS_LABEL);
         ashling_proc.setAttribute(IProcess.ATTR_CMDLINE, ash_cmd);
+        try {
```

Comment:
Comment in code and in commit description should explain why this additional "sleep" is required. I also think it should be in separate commit, becuse it is needed independently from the rest of commit.